lib/node/pl-config/src/index.ts

```diff
@@ -4,4 +4,4 @@ export * from './ports';
 export * from './license';
 export * from './auth';
 export * from './storages';
-export * from './package';
+export * from '../../pl-local/src/pl_version';
```

Comment:
Since we already made breaking changes to API in pl-config, this import also should be dropped